test/strategies/matic/multi-aave-mai-bal/MultiAMBDoHardWork.ts

```diff
@@ -50,9 +50,9 @@ export class MultiAaveMaiBalTest extends DoHardWorkLoopBase {
     const strategyAaveMaiBal: StrategyAaveMaiBal = this.strategy as StrategyAaveMaiBal;
 
     // claim aave rewards on mai
-    console.log('claimAaveRewards');
-    const cam = await DeployerUtils.connectInterface(this.signer, 'ICamToken', this.camToken) as ICamToken;
-    await cam.claimAaveRewards();
+    // console.log('claimAaveRewards');
+    // const cam = await DeployerUtils.connectInterface(this.signer, 'ICamToken', this.camToken) as ICamToken;
+    // await cam.claimAaveRewards(); // TODO !!! Warning: for some reason this not work now (after ICamToken contract upgrade or another reason)
```

Comment:
aave stop rewards, probably it is the reason

Reply:
how will it affect doHardWork process?

Reply:
It should not affect doHardWork - just tests.